robot/Cumulus/tests/browser/recurring_donations/zedit_effective_date_erd_record.robot

```diff
@@ -0,0 +1,127 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             DateTime
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable RD2
+#Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
+
+        ${effective_date_initial} =           Get Current Date      result_format=%-m/%-d/%Y
+        ${date} =           Get Current Date
+        ${DATE}=            Add Time To Date      ${date}  30 days      result_format=%Y-%m-%d %H:%M:%S.%f
+        ${date_to_update} =   Convert Date                   ${DATE}                         result_format=%Y-%m-%d
+        ${effective_modified_date}=        Add Time To Date      ${date}  0 days     result_format=%-m/%-d/%Y
+        ${currdate}=        Get Current Date  result_format=datetime
+        ${currentvalue} =   Evaluate          (${currdate.month-1}) * 100
+        ${currentvalue_edited}=   Evaluate    (${currdate.month}*100) + 150
+        Set Suite Variable  ${currentvalue}
+        Set Suite Variable  ${DATE}
+        Set Suite Variable  ${effective_modified_date}
+        Set Suite Variable  ${currentvalue_edited}
+        Set Suite Variable  ${effective_date_initial}
+        Set Suite Variable  ${date_to_update}
+
+        &{contact1_fields}=   Create Dictionary                     Email=rd2tester@example.com
+        &{recurringdonation_fields} =	Create Dictionary           Name=ERDTest1
+        ...                                                         npe03__Installment_Period__c=Monthly
+        ...                                                         npe03__Amount__c=100
+        ...                                                         npe03__Open_Ended_Status__c=${type}
+        ...                                                         Status__c=Active
+        ...                                                         Day_of_Month__c=15
+        ...                                                         InstallmentFrequency__c=${frequency}
+        ...                                                         PaymentMethod__c=${method}
+
+        Setupdata   contact         ${contact1_fields}             recurringdonation_data=${recurringdonation_fields}
+
+
+*** Variables ***
+${frequency}  1
+${day_of_month}  15
+${amount_to_update}  150
+${method}  Credit Card
+${type}    Open
```

Comment:
Same comment as above. Can you make it all caps?

Reply:
done

Reply:
So the variable declaration and also the reference needs to be in caps. I see that in your test case, you are still using it in small case.
Reasoning: When you look at the test case and see a variable being used, and if it is in all caps, it is implied that the variable must be declared elsewhere and it is global.

Reply:
Updated